peripherals/rcc/rcc_l1.yaml

```diff
@@ -0,0 +1,140 @@
+RCC:
+  CR:
+    RTCPRE:
+      DIV2: [0, "HSE divided by 2"]
+      DIV4: [1, "HSE divided by 4"]
+      DIV8: [2, "HSE divided by 8"]
+      DIV16: [3, "HSE divided by 16"]
+    "*ON":
+      Disabled: [0, "The selected clock is disabled"]
+      Enabled: [1, "The selected clock is enabled"]
+    PLLRDY:
+      _read:
+        Unlocked: [0, "PPL unlocked"]
+        Locked: [1, "PPL locked"]
+    HSEBYP:
+      NotBypassed: [0, "HSE oscillator not bypassed"]
+      Bypassed: [1, "HSE oscillator bypassed"]
+    "*RDY":
+      _read:
+        NotReadied: [0, "The selected oscillator is not stable"]
+        Readied: [1, "The selected oscillator is not stable"]

```

Comment:
These should be `NotReady` and `Ready` please. The description for `Ready` needs fixing too (it says "not stable").